data/insights-client.conf

```diff
@@ -54,3 +54,5 @@
 
 # Location of the tags file for this system
 #tags_file=/etc/insights-client/tags.yaml
+
+#ros_collect=True
```

Comment:
If we're going to include a default suggested value in the config file, it should mirror the actual internal value of this config option. Is this additional data collection really enabled by default?

Also, if we're going to include this in the default config file, it needs documentation explaining what it does. And lastly, it will need an entry in `insights-client.conf.5`, again defining what the option does.

Reply:
As per new approach the param is not included in file by default. Marking this resolved accordingly.